x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/audit/logfile/LoggingAuditTrail.java

```diff
@@ -443,7 +448,22 @@ public LoggingAuditTrail(Settings settings, ClusterService clusterService, Threa
     }
 
     @Override
-    public void authenticationSuccess(String requestId, Authentication authentication, RestRequest request) {
+    public void authenticationSuccess(RestRequest request) {
```

Comment:
This method must now be called in an authenticated context, when the HTTP request body is available.